diff --git a/dist/keystatic-core-ui.js b/dist/keystatic-core-ui.js
index b9e7dd88647f58af0aca3a401cb9850bd90e3003..495f7955dc2470ce95790c1865d9dfad9597e609 100644
--- a/dist/keystatic-core-ui.js
+++ b/dist/keystatic-core-ui.js
@@ -8478,7 +8478,9 @@ function Keystatic(props) {
   let t2;
   if ($[5] !== props.appSlug || $[6] !== t1) {
     t2 = /*#__PURE__*/jsx(ClientOnly, {
-      children: /*#__PURE__*/jsx(RedirectToLoopback, {
+      children: /*#__PURE__*/jsx(props.config.storage.kind === "local"
+        ? Fragment
+        : RedirectToLoopback, {
         children: /*#__PURE__*/jsx(AppSlugProvider, {
           value: props.appSlug,
           children: t1
